android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/bmicompute/BmiQuestionnaireViewModel.kt

```diff
@@ -183,18 +185,20 @@ class BmiQuestionnaireViewModel @Inject constructor(val patientRepository: Patie
     questionnaireResponse: QuestionnaireResponse,
     patientId: String,
     encounterID: String,
-    height: Double,
     weight: Double,
-    computedBMI: Double
+    height: Double,
+    computedBMI: Double,
+    isUnitModeMetric: Boolean
   ): Boolean {
     return patientRepository.recordComputedBmi(
       questionnaire,
       questionnaireResponse,
       patientId,
       encounterID,
-      height,
       weight,
-      computedBMI
+      height,
+      computedBMI,
+      isUnitModeMetric
```

Comment:
Add parameter names to improve readability and ease of code refactor

Reply:
done